lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -106,6 +111,7 @@ public void Submit(IJobSubmission jobSubmission)
                 javaParams.TcpPortRangeCount.ToString(),
                 javaParams.TcpPortRangeTryCount.ToString()
                 );
+            _jobStatus = State.SUBMITTED;
```

Comment:
This assumes that this instance of `IREEFClient` is used to submit only one job. This isn't how the interface was initially documented, and I don't think we need to make this assumption now. If this method is called more than once, `_subStatus` will be all messed up.